study.md

```diff
@@ -9,17 +9,37 @@ Read the following articles thoroughly. Practice active reading by taking notes.
 
 As you read, pay special attention to addressing the following:
 
--   Identify use cases where relational or non-relational databases are
-    appropriate.
--   List three strengths of relational databases.
--   List three strengths of non-relational databases.
+### Identify use cases where relational or non-relational databases are appropriate.
 
-After you've finished your study, summarize your notes to the following section
-and open a pull request.
+For websites that manage *big data* like Facebook, Google and Amazon, non-relational database is more appropriate due to its scalability and efficiency.
+
```

Comment:
This may or may not be correct. Usually social networks require relationships which makes a relation database better but there are always outliers or use cases for either.